x-pack/plugins/security_solution/public/common/components/matrix_histogram/index.tsx

```diff
@@ -180,12 +184,7 @@ export const MatrixHistogramComponent: React.FC<MatrixHistogramComponentProps> =
   };
   const [loading, { data, inspect, totalCount, refetch }] =
     useMatrixHistogramCombined(matrixHistogramRequest);
-  const [{ pageName }] = useRouteSpy();
-
-  const onHostOrNetworkOrUserPage =
-    pageName === SecurityPageName.hosts ||
-    pageName === SecurityPageName.network ||
-    pageName === SecurityPageName.users;
+  const onHostOrNetworkOrUserPage = isExplorePage(pathname);
```

Comment:
Nice! could we also change that var name to `const onExplorePage`? This way we won't have to come here to rename it if we add or remove pages in explore 😅